CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## v1.1.0 - 09/2025
```

Comment:
- smartphone container enrollment
- enroll_via_multichallenge is cancellable if enabled in privacyIDEA

Reply:
[afb3ebc](https://github.com/privacyidea/privacyidea-nextcloud-app/pull/26/commits/afb3ebce5ef9213fa235fa03508eee45f170fb4f)